tach.yml

```diff
@@ -1,107 +0,0 @@
-modules:
```

Comment:
This file should not be deleted, it is how we configure `tach` for our own repo.